src/julia.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 #include "libsupport.h"
 #include <stdint.h>
 #include <string.h>
+#include <inttypes.h>
```

Comment:
Done.

I hope llvm defines enough of them.... Otherwise I would have to manually define the missing ones (or try to use type casts).